Implement ES and FR translations

Sergiodero opened the pull request:
This PR adds translated files for ES (done) and FR (pending) translations for Elrond. 

Section | Page | Formatting | Correct Translations
-- | -- | -- | -- 
Getting started | index | ✅ | ⬜
Technical Doc Schedule | Reference | ✅ | ✅
Technical Doc Schedule | Best Practices | ✅ | ✅
Technical Doc Schedule | Data Examples | ✅ | ✅
Technical Doc Schedule | Change History | ✅ | ✅
Technical Doc Realtime | Reference | ✅ | ✅
Technical Doc Realtime | Best Practices | ✅ | ✅
Technical Doc Realtime | Data Examples | ✅ | ✅
Technical Doc Realtime | Change History | ✅ | ✅
Technical Doc Realtime | Language Bindings (FR and EN) | ✅ | ✅
Community | index | ✅ | ✅
About | index | ✅ | not doing
Navigation | mkdocs.yml  | ✅ | ✅

Sergiodero committed:
Add ES and FR config in mkdocs.yml

Sergiodero committed:
ES Getting Started section

Sergiodero committed:
ES Documentation Overview

Sergiodero committed:
ES Schedule Reference doc

Sergiodero committed:
ES Schedule Best Practices doc

Sergiodero committed:
FR What is GTFS? doc

Sergiodero committed:
Add ES assets folder

Sergiodero committed:
ES Schedule examples

Sergiodero committed:
ES change history

Sergiodero committed:
ES Realtime reference

Sergiodero committed:
ES Realtime best practices

Sergiodero committed:
ES Realtime Feed Entities

Sergiodero committed:
ES Realtime language bindings

Sergiodero committed:
ES Realtime examples

Sergiodero committed:
ES Realtime change history

Sergiodero committed:
ES fix Schedule reference links

Fixes two broken links in the Reference document for GTFS Schedule

Sergiodero committed:
ES Community section

Sergiodero committed:
ES About

Sergiodero committed:
ES Proto files

Sergiodero committed:
Update configs in mkdocs.yml & add nav in ES and FR

This commit adds the updated configs in mkdocs to conform with v1.0 of the plug-in. It also adds the nav translation for ES and sets up the structure for FR nav translation.

Sergiodero committed:
FR add rest of translated files (not reviewed)

Sergiodero committed:
FR Update "home" navigation translation in mkdocs.yml

Sergiodero committed:
FR Why use GTFS? page (Fix formatting)

Sergiodero committed:
FR Create page (Fix formatting)

Sergiodero committed:
FR Validate page (Fix formatting)

Sergiodero committed:
FR Publish page (Fix formatting)

Sergiodero committed:
FR base add-ons page (Fix formatting)

Sergiodero committed:
FR Base page (Fix formatting)

Sergiodero committed:
FR Accessibility page (Fix formatting)

Sergiodero committed:
FR Fares page (Fix formatting)

Sergiodero committed:
FR Small fix to accessibility page translation

Sergiodero committed:
FR Pathways page (Fix formatting)

Sergiodero committed:
FR Flexible services page (Fix formatting)

Sergiodero committed:
FR Features Overview page (Fix formatting)

eliasmbd committed:
Format Review of FR Community Section

- Checked and corrected format of all FR pages in Community section.
- Made minor corrections to ENG version of Flex

eliasmbd commented:
Format Review of FR Community Section DONE!

eliasmbd committed:
Formatting Reference.md Schedule FR

- Checked Technical Documentation Overview
- Formatted GTFS Schedule Reference

eliasmbd commented:
- Checked Technical Documentation Overview
- Formatted GTFS Schedule Reference

eliasmbd committed:
Update schedule_best_practices.md

eliasmbd commented:
Formatted GTFS Schedule best Practices FR

eliasmbd committed:
Formatting Data Examples Part 1 FR

- Attributions.md
- continuous-stops.md
-fares-v1.md
- Fares-v2.md
- Flex.md

richfab committed:
FR Community Get Involved (Correct Translations)

richfab committed:
FR Community Fares v2 (Correct Translations)

richfab committed:
FR Community Flex (Correct Translations)

richfab committed:
FR Community Reatime Amendment (Correct Translations)

richfab committed:
FR Community Schedule Amendment (Correct Translations)

richfab committed:
FR Community Overview (Correct Translations)

richfab committed:
FR Community Flex and Realtime Amendment (Fix Anchors)

richfab commented:
Correct Translations of FR Community Section DONE ✅

eliasmbd committed:
Formatting Schedule examples FR part 2

Completed formatting changes of the rest of the schedule examples in the technical documentation.

eliasmbd committed:
Merge branch 'ES-and-FR-translation-implementation-20240718' of https://github.com/MobilityData/elrond.gtfs.org into ES-and-FR-translation-implementation-20240718

eliasmbd committed:
Formatting Change History Schedule FR

eliasmbd commented:
@richfab Technical Doc Schedule section ready for translations

eliasmbd committed:
Formatting Reference Realtime FR

eliasmbd committed:
Formatting Best Practices Realtime FR

richfab committed:
FR Documentation Schedule Reference (Correct Translations) WIP

eliasmbd committed:
Formatting Feed Entities Realtime FR

eliasmbd committed:
Formatted Language Binding Realtime EN and FR

Fixed random line changes in both languages. Translation might be off

richfab committed:
FR Documentation Schedule Reference (Correct Translations) WIP

richfab committed:
Merge branch 'ES-and-FR-translation-implementation-20240718' of https://github.com/MobilityData/elrond.gtfs.org into ES-and-FR-translation-implementation-20240718

eliasmbd committed:
Formatted Data Examples Realtime FR

eliasmbd committed:
Merge branch 'ES-and-FR-translation-implementation-20240718' of https://github.com/MobilityData/elrond.gtfs.org into ES-and-FR-translation-implementation-20240718

eliasmbd committed:
Formatting Change History Realtime FR

eliasmbd committed:
Formatting About FR

eliasmbd commented:
@richfab 
- Formatting of Realtime except Protobuf in done
- Formatting of About page is done

eliasmbd committed:
Updated menu fields FR

Will require revision due to missing fields and possible inconsistencies with other sections

richfab commented:
Here is a list of missed translations that I have observed, if it helps.

Eligible for systematic translation:

Incorrect translation | Correct translation | English source
-- | -- | -- 
proposant, avocat, défenseur | _advocate_ (written in English in italic) | advocate
demande de tirage, demande d’extraction, demandes de tirage | pull request | pull request
ID unique | Clé primaire | Unique ID
ID étranger, Carte d’identité étrangère | Clé étrangère | Foreign ID
ensemble de données | jeu de données | dataset
alimentation, aliments pour animaux |  flux | feed
un application | une application | an application
analyser un flux | lire un flux | to parse a feed
cycliste, coureur, motard | passager  | rider
sentier | passage | path
planificateurs de voyage | planificateurs d’itinéraire | trip planner
planification de voyage | planification d'itinéraire | trip planning
temps réel GTFS, GTFS en temps réel | GTFS Realtime | GTFS Realtime
calendrier GTFS | GTFS Schedule | GTFS Schedule
Parcours GTFS | GTFS-Pathways | GTFS-Pathways
GTFS-Occupations | GTFS-Occupancies | GTFS-Occupancies
Éligibilités GTFS | GTFS-Eligibilities | GTFS-Eligibilities
Capacités GTFS | GTFS-Capabilities | GTFS-Capabilities
sièges GTFS | GTFS-seats | GTFS-seats
Tarifs v1 | Fares v1 | Fares v1
Tarifs v2 | Fares v2 | Fares v2
spécifications@mobilitydata.org | specifications@mobilitydata.org (no accent) | specifications@mobilitydata.org
meilleures pratiques, recommandations pratiques | bonnes pratiques | best practices
haltes pour enfants, arrêts pour enfants, arrêt enfant | arrêt sous-jacent | child stop
station mère | station principale | parent station
succursale | branche | branch
Exemple de source | Source de l'exemple | Source example
Apple Payer | Apple Pay | Apple Pay
services de numérotation | transport à la demande | dial-a-ride

Require manual revision (not eligible for systematic translation):

Incorrect translation | Correct translation | English source
-- | -- | -- 
numéro, problème | issue | issue
engagements | commits | commits
fusionnées | mergées | merged
enregistrement | entrée (a row in a table) | record
domaine | champ (a column in a table) | field
analyseurs | lecteurs de flux | parsers
panneau de signalisation, phare | girouette | headsign
retour | dépôt | drop-off
étape | trajet | leg
délais | périodes horaires | timeframes
Beaucoup | Plusieurs | Many (cardinality)

fredericsimard committed:
Added favicon

Sergiodero committed:
EN update schedule reference doc

This change is to include:

- Updated revision date
- PR #479
- PR #460
- PR #470
- PR #474
- PR #472
- PR #485

Sergiodero commented:
FYI I added a new commit that updates the EN schedule reference document, I will do the same thing for ES and FR. Also the Schedule revision history will be updated.

Sergiodero committed:
EN update revision history for Schedule

This update pair the revision history with the latest version in google/transit, updated in PR # 491

Sergiodero committed:
EN fix small typo in RT trip updates as per PR # 486

Sergiodero committed:
Add ES & FR translated images for Getting started section

Sergiodero committed:
ES Update Getting Started nav translation

richfab committed:
FR Documentation Schedule Best Practices (Correct Translations)

richfab committed:
FR Documentation Schedule Examples (Correct Translations) WIP

Sergiodero committed:
ES update Schedule reference doc

This change is to include:

- Updated revision date
- PR  # 479
- PR  # 460
- PR  # 470
- PR  # 474
- PR  # 472
- PR  # 485

Sergiodero committed:
ES fix typo in Schedule Ref doc

Sergiodero committed:
ES update revision history for Schedule

This update pair the revision history with the latest version in google/transit, updated in PR # 491

Sergiodero committed:
EN fix formatting heading issue for Schedule revision history

Sergiodero committed:
FR update revision history for Schedule

This update pair the revision history with the latest version in google/transit, updated in PR # 491

Sergiodero committed:
FR update schedule reference doc

This change is to include:

- Updated revision date
- PR  #479
- PR  # 460
- PR  # 470
- PR  # 474
- PR  # 472
- PR  # 485

fredericsimard committed:
Added Blog to the nav menu + redirect

... will redirect to blog.gtfs.org

fredericsimard committed:
Added 2 more redirections for blog

richfab committed:
FR Documentation Schedule Examples (Correct Translations) WIP

richfab committed:
FR Documentation Schedule Examples (Correct Translations) WIP

Sergiodero committed:
ES Fix broken anchors

fredericsimard committed:
Removed `!important` flag at Alessandro's request

fredericsimard committed:
Merge branch 'ES-and-FR-translation-implementation-20240718' of https://github.com/MobilityData/elrond.gtfs.org into ES-and-FR-translation-implementation-20240718

Sergiodero committed:
FR fix broken anchors

fredericsimard committed:
Added missing translation page to all missing translations

... this is a dummy text for now, will be translated soon.

fredericsimard committed:
Fixed 404 and added all languages to it, +responsive

fredericsimard committed:
Removed individual localized 404 files

fredericsimard committed:
Switched RU and JA in 404

fredericsimard committed:
Changed Japanese

richfab committed:
FR Documentation Schedule Examples (Correct Translations)

fredericsimard committed:
Merge branch 'main' into ES-and-FR-translation-implementation-20240718

richfab committed:
FR Navigation (Correct Translations)

richfab committed:
FR Documentation Realtime (Correct Translations)

fredericsimard reviewed (approved):
Looks good, crossing fingers! 🤞

isabelle-dr commented on docs/fr/documentation/schedule/reference.md:
```diff
@@ -111,7 +111,7 @@
  |------|------|------| 
  | [agency.txt](#agencytxt) | **Requis** | Agences de transport en commun dont le service est représenté dans ce jeu de données. | 
- | [stops.txt](#stopstxt) | **Requis** | Arrêts où les véhicules prennent ou déposent les passagers. Définit également les gares et les entrées des gares. | 
+ | [stops.txt](#stopstxt) | **Requis sous condition** | Arrêts où les véhicules prennent ou déposent les passagers. Définit également les gares et les entrées des gares. <br><br>Requis sous condition :<br> - Optionnel si des zones à la demande sont définies dans [locations.geojson](#locationsgeojson). <br>- **Obligatoire** sinon. | 
```
I find this slightly confusing but the confusion comes from the English (_Optional if demand-responsive zones are defined in locations.geojson_), not the translation. The demand-responsive applies to a **service**, not a zone. It would be clearer if it was:

> Optionnel si des services de transport à la demande zonaux sont définis dans [...]

Which would need the English to be:

> Optional if zonal demand-responsive transit services are defined in [...]

Sergiodero replied:
Got it! I'm leaning towards changing only the FR translation in this case as the english adjustment seems to me also not ideal.

isabelle-dr commented on docs/fr/documentation/schedule/reference.md:
```diff
@@ -250,7 +250,7 @@
  | `service_id` | Clé étrangère faisant référence à `calendar.service_id` ou `calendar_dates.service_id` | **Requis** | Identifie un ensemble de dates auxquelles le service est disponible pour un ou plusieurs itinéraires. | 
  | `trip_id` | Identifiant unique | **Requis** | Identifie un voyage. | 
- | `trip_headsign` | Text | Optionnel | Text qui apparaît sur la signalisation identifiant la destination du voyage aux passagers. Doit être utilisé pour distinguer les différents modèles de service sur le même itinéraire.<br><br> Si le signe change pendant un voyage, les valeurs de `trip_headsign` peuvent être remplacées en définissant des valeurs dans `stop_times.stop_headsign` pour des `stop_t ...
+ | `trip_headsign` | Text | Optionnel | Text qui apparaît sur la signalisation identifiant la destination du voyage aux passagers. Ce champ est recommandé pour tous les services avec une Girouette affichée sur le véhicule qui peut être utilisé pour distinguer les trajets d'un itinéraire.<br><br> Si le signe change pendant un voyage, les valeurs de `trip_headsign` peuvent être remplacées en défini ...
```
1. _Text_ should be _Texte_
2. _Girouette_ should be _girouette_
3. _Si le signe change_ should be _Si la girouette change_

Sergiodero replied:
I updated the text to reflect these corrections, including changing all mentions of "text" in the doc to "texte"

isabelle-dr commented on docs/fr/documentation/schedule/reference.md:
```diff
@@ -582,7 +582,7 @@
  Clé primaire (`shape_id`, `shape_pt_sequence`) 
  
- Les Tracé des lignes décrivent le chemin qu’un véhicule parcourt le long d’un l’alignement de l’itinéraire, et sont définis dans le fichier shapes.txt. Les Tracé des lignes sont associées aux Trajets et consistent en une séquence de points par lesquels le véhicule passe dans l’ordre. Les Tracé des lignes n’ont pas besoin d’intercepter exactement l’emplacement des Arrêts, mais tous les Arrêts d’u ...
+ Les Tracé des lignes décrivent le chemin qu’un véhicule parcourt le long d’un l’alignement de l’itinéraire, et sont définis dans le fichier shapes.txt. Les Tracé des lignes sont associées aux Trajets et consistent en une séquence de points par lesquels le véhicule passe dans l’ordre. Les Tracé des lignes n’ont pas besoin d’intercepter exactement l’emplacement des Arrêts, mais tous les Arrêts d’u ...
```
Here the English is:

> The shapes.txt file should be included for all route-based services (not for zone-based demand-responsive services).

I'm surprised the _should_ got translated to _dois_ (which is must in _English_). 
In the glossary, _should_ should be translated into french to _devrait_, which would give this translation:

> Le fichier shapes.txt **devrait** être inclus [...]

Sergiodero replied:
I think this is an isolated case, because I translated these changes manually (without the glossary taking effect). Thanks for flagging it!

isabelle-dr reviewed (commented):
@Sergiodero I provided comments inline. 
I have a meta comment also: in GTFS, we have a distinction between an ID and a primary key. 
An ID can be:
- _ID_ (assumed non-unique)
- _Unique ID_
- _Foreign ID_
A _Primary key_ is the field(s) that are distinctive for each record. It's often a _Unique ID_ but not always. 
We don't have anything just called "key" in GTFS.

This distinction is unclear in French.
<img width="521" alt="Screenshot 2024-08-26 at 4 18 47 PM" src="https://github.com/user-attachments/assets/2726e50b-f8f4-4b5a-9f3e-1aa6842def23">

<img width="518" alt="Screenshot 2024-08-26 at 4 19 06 PM" src="https://github.com/user-attachments/assets/f0376bfe-34f3-4d42-a18d-a2eaa13d9747">
<img width="387" alt="Screenshot 2024-08-26 at 4 26 11 PM" src="https://github.com/user-attachments/assets/5e0ec3f2-57ef-4f8c-b973-8f52ede9b8e7">


I suggest we do the following:
- translate _ID_ as **ID** (this is already the case)
- translate _Unique ID_ as **ID unique**
- translate _Foreign ID_ as **ID étranger**
- translate _Primary key_ as **Clé primaire** (this is already the case)

Sergiodero commented:
Thanks @isabelle-dr! I've updated the text to address both your specific and meta comments. these changes are included in PR #116
